### DISCUSSION TITLE
Cannot deploy on Render

### DISCUSSION THREAD

**Discussion by 0x090909** [fief-dev/fief#358] [Q&A]
Hello, Im facing a problem while deploying on render. 

I get the following error: 

`fief migrate: unknown command`

the command comes from the `pre-deploy.sh`

Can you help ?

Also while looking at the possible commands for fief, there is no mention of migrate anywhere 

Thank you

**Comment by fief-bailiff** [comment]
Hail, @0x090909 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by maximgx** [comment]
Same situation for `fief run-worker` command

**Comment by frankie567** [comment]
I've updated the Render blueprint to use the latest version of Fief, so it should work now.

During my tests, I've found that there is a [bug in Render](https://community.render.com/t/can-not-apply-blueprints-graphql-returns-422/3203), which prevent the blueprint to create if you fill the variables prompted by the system. If you are impacted, you should leave them blank and add them manually in your environment group.

Note that if during deploy or startup, you have an `unknown command`, it probably means a setting is missing (see #356 for more background about this).

**Reply by maximgx** [reply]
Now it works,

However, when I
 - deploy from blueprint it obviously doesn't work because there are missing env var;
 - then I launch `fief quickstart` to obtain some of them and insert my mail/password;
 - the server successfully goes live, but by going to `<deploy_link>/admin` it redirects me to `localhost:8000`;
 
Am I missing something here? Note that I **didn't** copy the host env var localhost:8000

**Reply by frankie567** [reply]
You probably need to set `FIEF_DOMAIN` variable, e.g. `fief.mydomain.com`

**Reply by maximgx** [reply]
Apparently that was the problem, although had to try more than one re-deploys
